examples/README.md

```diff
@@ -106,30 +106,30 @@ When you open a notebook, select the kernel `nvflare_example` using the dropdown
-| [Federated Learning with Differential Privacy for BraTS18 segmentation](./advanced/brats18/README.md)                                  | MONAI        | -                                                   | see requirements.txt                      | Illustrates the use of differential privacy for training brain tumor segmentation models using federated learning.                                              |
-| [Federated Learning for Prostate Segmentation from Multi-source Data](./advanced/prostate/README.md)                                   | MONAI        | -                                                   | see requirements.txt                      | Example of training a multi-institutional prostate segmentation model using [FedAvg](https://arxiv.org/abs/1602.05629), [FedProx](https://arxiv.org/abs/1812.06127), and [Ditto](https://arxiv.org/abs/2012.04221). |
+| Example                                                                                                                                | Framework    | Notes                                     | Summary                                                                                                                                                         |
+|----------------------------------------------------------------------------------------------------------------------------------------|--------------|-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [NVFlare + MONAI integration](../integration/monai/README.md)                                                                          | MONAI        | -                                         | For an example of using NVIDIA FLARE to train a 3D medical image analysis model using federated averaging (FedAvg) and MONAI Bundle, see [here](../integration/monai/examples/README.md). |
+| [Federated Learning with Differential Privacy for BraTS18 segmentation](./advanced/brats18/README.md)                                  | MONAI        | see requirements.txt                      | Illustrates the use of differential privacy for training brain tumor segmentation models using federated learning.                                              |
+| [Federated Learning for Prostate Segmentation from Multi-source Data](./advanced/prostate/README.md)                                   | MONAI        | see requirements.txt                      | Example of training a multi-institutional prostate segmentation model using [FedAvg](https://arxiv.org/abs/1602.05629), [FedProx](https://arxiv.org/abs/1812.06127), and [Ditto](https://arxiv.org/abs/2012.04221). |
```

Comment:
Do we need the "Notes" column?
it only said "see requirements.txt"
but nothing other than dependencies is there

Reply:
In general, adding columns in the beginning was to make the table appear more like a table, since two or even three did not feel to me like enough to warrant the format of a table. So to me, the need of these columns is a balance of that with the possibility of us using these columns.

Reply:
@chesterxgchen can you comment on do we want this here?
I guess it is more of subjective preference, we can ask product or our users.